### PR TITLE
fix: error on camera permissions denied

### LINKED ISSUE #600: Throw error when denying camera permissions on mobile
Currently this is the result clicking "Scan QR Code" and then denying camera permissions. The X is hard to find to even unblock the web app again, it should throw a "Camera Permissiosn Denied" error and the web app should continue working and prssing "Scan QR Code" button should throw this error once permissions are denied. 

image

### PR DESCRIPTION
**Pull request by dni**
closes #600

@kilrau translations

### CODE REVIEW

File: src/i18n/i18n.ts [hunk 1]
```diff
@@ -201,6 +201,7 @@ const dict = {
         reckless_mode_setting: "Reckless Mode",
         reckless_mode_setting_tooltip:
             "Disables prompts to download refund file and other confirmation steps",
+        error_starting_qr_scanner: "Error starting QR scanner",
```
Comment (michael1011) [inline src/i18n/i18n.ts:204]: How about asking to check the permissions?

Comment (dni) [inline src/i18n/i18n.ts:204]:
```suggestion
        error_starting_qr_scanner: "Error starting QR scanner, check you camera permissions!",
```

**Review by kilrau — CHANGES_REQUESTED** [review]
Strings added and tested - looking good. What I do not like is that the "Scan QR Code" button disappears after a failed permissions. One has to refresh the site to get a new try. Just keep the button there please.
[Screencast from 07-16-2024 11:39:08 AM.webm](https://github.com/user-attachments/assets/9b476d8b-7d3d-4ee8-9a08-1c98b524dd56)

**Comment by dni** [comment]
> Strings added and tested - looking good. What I do not like is that the "Scan QR Code" button disappears after a failed permission request. One has to refresh the site to get a new try. Just keep the button there please. [Screencast from 07-16-2024 11:39:08 AM.webm](https://github.com/user-attachments/assets/9b476d8b-7d3d-4ee8-9a08-1c98b524dd56)

done

**Review by kilrau — APPROVED** [review]
"Scan QR" button doesn't disappear now. Error comes up every time I press it. Behaves as expected.

**Review by michael1011 — APPROVED** [review]

**Pull request merged**